packages/theming/atlas/src/themesource/atlas_core/web/main.scss

```diff
@@ -1,90 +1,138 @@
 @import "core/base/mixins/buttons";
 @import "core/base/mixins/groupbox";
-@import "core/base/animation";
-@import "core/base/flex";
-@import "core/base/spacing";
-@import "core/base/reset";
-@import "core/base/base";
-@import "core/base/login";
+@include include-styles("core/base/animation", $exclude-base);
+@include include-styles("core/base/flex", $exclude-base);
+@include include-styles("core/base/spacing", $exclude-base);
+@include include-styles("core/base/reset", $exclude-base);
+@include include-styles("core/base/base", $exclude-base);
+@include include-styles("core/base/login", $exclude-base);
 
 // Widgets
-@import "core/widgets/inputs";
-@import "core/helpers/backgrounds";
-@import "core/widgets/labels";
-@import "core/widgets/badge";
-@import "core/helpers/labels";
-@import "core/widgets/buttons";
-@import "core/helpers/buttons";
-@import "core/widgets/checkbox";
-@import "core/widgets/grid";
-@import "core/widgets/datagrids";
-@import "core/helpers/datagrids";
-@import "core/widgets/dataview";
-@import "core/widgets/datepicker";
-@import "core/widgets/header";
-@import "core/widgets/glyphicons";
-@import "core/widgets/groupbox";
-@import "core/helpers/groupbox";
-@import "core/helpers/images";
-@import "core/widgets/listview";
-@import "core/helpers/listview";
-@import "core/widgets/modals";
-@import "core/widgets/navigationbar";
-@import "core/helpers/navigationbar";
-@import "core/widgets/navigationlist";
-@import "core/widgets/navigationtree";
-@import "core/helpers/navigationtree";
-@import "core/widgets/popupmenu";
-@import "core/widgets/simplemenubar";
-@import "core/helpers/simplemenubar";
-@import "core/widgets/radiobuttons";
-@import "core/widgets/tabcontainer";
-@import "core/helpers/tabcontainer";
-@import "core/widgets/tables";
-@import "core/helpers/tables";
-@import "core/widgets/templategrids";
-@import "core/helpers/templategrids";
-@import "core/widgets/typography";
-@import "core/helpers/typography";
-@import "core/widgets/layoutgrid";
-@import "core/widgets/pagination";
-@import "core/widgets/progress";
-@import "core/widgets/progressbar";
-@import "core/helpers/progressbar";
-@import "core/widgets/progresscircle";
-@import "core/helpers/progresscircle";
-@import "core/widgets/rating";
-@import "core/helpers/rating";
-@import "core/widgets/rangeslider";
-@import "core/helpers/rangeslider";
-@import "core/widgets/slider";
-@import "core/helpers/slider";
-@import "core/widgets/timeline";
-@import "core/helpers/helperclasses";
+@include include-widget-styles("core/widgets/input", $exclude-input);
+@include include-styles("core/helpers/background", $exclude-background-helpers);
+@include include-widget-styles("core/widgets/label", $exclude-label);
+@include include-widget-styles("core/widgets/badge", $exclude-badge);
+@include include-styles("core/helpers/label", $exclude-label-helpers);
+@include include-widget-styles("core/widgets/button", $exclude-button, "core/helpers/button", $exclude-button-helpers);
```

Comment:
Howcome you made the widget & helper styles singular? This is not a hard request, but I'd like to see this applied everywhere at some point if this is a rule-of-thumb, e.g. [1](https://github.com/mendix/widgets-resources/pull/507/files#diff-220405e6b901df6ad6e32c47cd4106a34526c449cc15d628eb1fb4884bc0c4c2R17) & [2](https://github.com/mendix/widgets-resources/pull/507/files#diff-220405e6b901df6ad6e32c47cd4106a34526c449cc15d628eb1fb4884bc0c4c2R20)

Reply:
What do you mean exactly?

Reply:
I see you change a lot of plural to singular: buttons -> button; labels -> label. 

[Here's an instance of plural case](https://github.com/mendix/widgets-resources/pull/507/files#diff-948d003df2838c3991976b6aadc120a7e1ea425d38182d2475d8b2b087faffa4R10): animation**S** that I think should be singular to keep consistent.

Reply:
What are your thoughts and do you want to address here if you agree?

Reply:
In native the variables look a bit different
- excludeActivityIndicatorDefaultClasses
- excludeActivityIndicatorHelperClasses

Maybe better to align?
- $exclude-label-default-classes
- $exclude-label-helder-classes

Check https://github.com/mendix/widgets-resources/pull/494/files for full reference.

Reply:
Resolved according to alignment.